src/Puppeteer.js

```diff
@@ -34,6 +35,7 @@ module.exports = class {
     this._isPuppeteerCore = isPuppeteerCore;
     // track changes to Launcher configuration via options or environment variables
     this.__productName = productName;
+    this._customQueryFunctions = new Map();
```

Comment:
One last thing, let's do a global rename for "queryFunction" to "queryHandler" so that the naming is internally consistent

Reply:
Actually, that Map isn't necessary. Removed.